README.md

```diff
@@ -1,65 +1,19 @@
-# Welcome to Remix!
+# Netlify Remix Template
 
-This template uses the Netlify Edge Functions beta to deploy your Remix site.
-
-<a href="https://app.netlify.com/start/deploy?repository=https://github.com/netlify/remix-edge-template"><img src="https://www.netlify.com/img/deploy/button.svg" alt="Deploy to Netlify"></a>
-
-- [Remix Docs](https://remix.run/docs)
-
-## Netlify Setup
-
-1. Install the [Netlify CLI](https://www.netlify.com/products/dev/):
-
-```sh
-npm i -g netlify-cli
-```
-
-If you have previously installed the Netlify CLI, you should update it to the latest version:
-
-```sh
-npm i -g netlify-cli@latest
-```
-
-2. Sign up and log in to Netlify:
-
-```sh
-netlify login
-```
-
-3. Create a new site:
-
-```sh
-netlify init
-```
+Welcome to the Netlify Remix Template project.
```

Comment:
Maybe here would be a good place to give a brief summary what the project is, like in the original README. Something like: 
```suggestion
Welcome to the Netlify Remix Template project.

This project includes:
- Netlify Functions template for Remix sites
- Netlify Edge Functions template for Remix sites
```
Or whatever they're named, just to make it clear both templates are included in this repo.